src/main/kotlin/org/malachite/estella/ApplicationProperties.kt

```diff
@@ -0,0 +1,57 @@
+package org.malachite.estella
+
+import com.beust.klaxon.Klaxon
+import java.io.File
+
+const val applicationPath = "src/main/resources/application.properties"
+const val configPath = "config2.json"
+
+fun getConfigurationData(): MutableMap<String, String> {
+    val file = File(configPath)
+    if (!file.exists())
+        return HashMap()
+    return Klaxon().parse<MutableMap<String, String>>(File(configPath)) ?: HashMap<String, String>()
+}
+
+fun getApplicationPropertiesForSql(env: MutableMap<String, String>): String {
+    return """
+			spring.datasource.url=${env["DATABASE_URL"]}
+            spring.jpa.generate-ddl=true
+            spring.jpa.hibernate.ddl-auto=create
+			spring.jpa.show-sql=true
+			spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+			spring.datasource.driver-class-name=org.postgresql.Driver
+		""".trimIndent()
+}
+
+fun getApplicationPropertiesForH2(): String {
+    return """
+        spring.datasource.url=jdbc:h2:file:./myDB
+        spring.datasource.username=admin
+        spring.datasource.password=admin
+        spring.jpa.generate-ddl=true
+        spring.jpa.hibernate.ddl-auto=create
+        spring.jpa.show-sql=true
+        spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+        spring.datasource.driver-class-name=org.h2.Driver
+        spring.datasource.driverClassName=org.h2.Driver
+        spring.h2.console.enabled=true
+        springdoc.swagger-ui.path=/docs
+    """.trimIndent()
+}
+
+fun prepareSpringProperties(env: MutableMap<String, String> = getConfigurationData()) {
+    var properties = if (env.isNotEmpty()) getApplicationPropertiesForSql(env) else getApplicationPropertiesForH2()
+    properties += "\n" + getOtherApplicationProperties()
+    File(applicationPath).printWriter().use { out ->
+        out.println(properties)
+    }
+}
+
+fun getApiKey():String=
+    System.getenv().getOrDefault("API_KEY","API_KEY")
```

Comment:
na heroku już takowy dodałeś?

Reply:
Póki co nie, ale to w każdej chwili będzie można zmienić i się przeładuje heroku, więc wydaje mi się że teraz do developerki wystarczy nam API_KEY XD